tests/server/3-document.spec.js

```diff
@@ -208,5 +191,19 @@ describe('Document', () => {
               res.body.message.should.equal('Document not found');
               done();
           });
-    })
-})
+    });
+    
+    it('should return all regular documents if role=regular query is used', (done) => {
+        server
+          .get('/api/documents?role=regular')
+          .set('authorization', userToken)
+          .end((err, res) => {
+              console.log(res.body);
+              res.status.should.equal(200);
+              for(let i=0; i<res.body.length; i++) {
```

Comment:
You conveniently ignored this feedback in your latest PR #14 . Look here for more clarity on the subject and rectify accordingly. [Airbnb Rule - Keep it functional](https://github.com/airbnb/javascript#iterators--nope)